chapters/core/guides/index.rst

```diff
@@ -37,3 +37,4 @@ Further information on any specific method can be obtained in the :doc:`../api_r
    ../../evaluation/index
    ../../incremental_indexing/index
    ../../stress/index
+   ../../mime-type/index
```

Comment:
Can you actually just add this a page inside of the chapters folder directly? Not within its own folder? We are trying to flatten our folder structure for better SEO. 

so `chapters/mime_types.rst`

Reply:
nice idea! having so many index.htmls is not so good. you are right

Reply:
@FionnD What would you think of changing all file paths in one run instead of changing them one after another?
Therefore, we would avoid inconsistencies.

Reply:
My thoughts exactly! A community member is in the process of doing that in this [PR](https://github.com/jina-ai/docs/pull/98). So I think we should leave it to him and not interfere